2-mandatory/8-password-validator.js

```diff
@@ -23,7 +23,21 @@ PasswordValidationResult=  [false, false, false, false, true]
 
 */
 
-function validatePasswords(passwords) {}
+function validatePasswords(passwords) {
+  let duplicatesArray = passwords.map((e, i, a) => a.indexOf(e) === i)
+  let otherArray = passwords.map(i=>containsUppercaseLetter(i) && containsNumber(i) && containsLowercaseLetter(i)
+  && containsSymbol(i) && i.length>=5)
+  let resultArray=[]
+  for (let j =0; j<passwords.length; j++){
+      if(duplicatesArray[j]===otherArray[j]){
```

Comment:
```
if(true){
true}else{
false
}
```
could be simplified because the return value is a bolean.
```
duplicatesArray.map((e, i, a) => e == otherArray[I])
```

Reply:
thank you for helping me to shortening it